packages/plugin-di/src/_examples/sampleEvalutor.ts

```
@@ -0,0 +1,63 @@
+@injectable()
+export class SampleEvaluator extends BaseInjactableEvaluator {
+    constructor() {
+        super(options);
+    }
+
+    async handler(runtime: IAgentRuntime, memory: Memory, _state?: State) {
+        // Evaluation logic for the evaluator
+        elizaLogger.log("Evaluating data in sampleEvaluator...");
+
+        // Example evaluation logic
+        if (memory.content && memory.content.text.includes("important")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)